### ISSUE TITLE
Missing partition table on 20230920.iso

### ISSUE THREAD

**Issue by hcal** [Vanilla-OS/iso-builder#244]
### Issue Description

When burning the latest image (20230920) to a usb drive using Etcher, as directed in the handbook, the following error pops up when selecting the iso.  

```
Missing partition table
It looks like this is not a bootable image.  The image does not appear to contain a partition table.
```

I continued anyway and as the error message implies, my Huawei laptop does not recognize the boot disk.  

I have the same issue with other methods of writing the disk image including my standard go-to of Gnome Disks and dd.

### Steps to Reproduce

Write the image to disk
Insert USB in laptop
Boot holding F12
Boot menu appears but the disk with Vanilla OS isn't an option

### On what version of Vanilla OS this happens?

Unreleased

**Comment by hklemp** [comment]
I have the same problem, but I was also able to reproduce the behaviour with older builds. 

Update: 
On discord I found this note "For bare metal... just UEFI". With this tool https://unetbootin.github.io/ I was able to create a bootable USB stick on a Mac.

**Comment by hcal** [comment]
I can confirm that unetbootin solved the problem for me as well so I am able to run and test the alpha.  I assume that the goal isn't to require it to make a bootable iso, so I think its still a bug worth reporting.

**Comment by hklemp** [comment]
If you change the configuration under etc/auto/config in line 39 and set the parameter "binary-images" to "iso-hybrid", you get an iso that you can also boot via a USB stick. But I don't know what side effects this change has.

**Comment by taukakao** [comment]
@hklemp Thanks, from the documentation in https://manned.org/lb_config.1 this seems like the correct solution. 

(Sorry for the late reply)

**Comment by kbdharun** [comment]
Nice catch @hklemp, seems like we indeed need to set it to `iso-hybrid` for it to work with USB images (`iso` would only work with CDs and VMs), I started a test build at https://github.com/kbdharun/vanilla-os/actions/runs/7291091081 if all goes good when testing locally with Etcher and Rufus. I will update the same here too.

**Comment by taukakao** [comment]
Fixed in #281 

Balena Etcher still complains about it but now you can click on continue and the installation works fine.
